workflow/urls.py

```diff
@@ -21,14 +21,21 @@
     QuantitativeOutputsUpdate, QuantitativeOutputsDelete, BudgetList, BudgetCreate,
     BudgetUpdate, BudgetDelete, Report, SiteProfileObjects, checklist_update_link, delete_contact,
     ProfileTypeCreate, ProfileTypeList, ProfileTypeUpdate, ProfileTypeDelete,
-    FundCodeList, FundCodeCreate, FundCodeUpdate, FundCodeDelete, OfficeView, StakeholderTypeView, OrganizationView
+    FundCodeList, FundCodeCreate, FundCodeUpdate, FundCodeDelete, OfficeView, StakeholderTypeView, OrganizationView, 
+    ProgramView
 )
 from django.urls import path, re_path
 
 # place app url patterns here
 
 urlpatterns = [
     # level1
+    re_path(
+          r'level1_program/(?P<pk>.*)',
```

Comment:
Since we can change the text `program` from the configurations, I would rather remove `_program` from url.

Reply:
I added `level1_program` instead of `level1` since there's already an existing url called `level1` and I didn't want to break anything else.. what do you think?

Reply:
hmm, okay then, let's take a note to fix it later at some point.

Reply:
cool! i added it to this card ACT-658 in the backlog